backend/src/main/java/de/neuefische/backend/model/recipe/Recipe.java

```diff
@@ -0,0 +1,27 @@
+package de.neuefische.backend.model.recipe;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(collection = "recipes")
```

Comment:
You don't need the assignment "coolection=" I guess :) 
```suggestion
@Document("recipes")
```